src/runtime/micro/host_driven/utvm_runtime.c

```diff
@@ -101,7 +101,7 @@ void UTVMMain() {
 
 // We use a dummy function to signal execution is finished for device
 // backends which require breakpoints.
-void __attribute__((noinline)) UTVMDone() { utvm_done = 1; }
+void __attribute__((noinline,noreturn)) UTVMDone() { utvm_done = 1; for (;;) ; }
```

Comment:
Would we want this to trap ever in which case a __builtin_trap might be a trick to play ?

Reply:
I think at the moment, since we aren't installing a predictable ISR vector table, we should avoid triggering those. but we can revisit approaches like this when we switch to on-device runtime.

Reply:
fair enough, lgtm.